src/EditorFeatures/Core/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -20,13 +21,21 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.AsyncComplet
 {
     internal sealed partial class ItemManager : IAsyncCompletionItemManager2
     {
+        /// <summary>
+        /// The threshold for us to consider exclude (potentially large amount of) expanded items from completion list.
+        /// Showing a large amount of expanded items to user would introduce noise and render the list too long to browse.
+        /// Not processing those expanded items also has perf benefit (e.g. matching and highlighting could be expensive.)
+        /// We set it to 2 because it's common to use filter of length 2 for camel case match, e.g. `AB` for `ArrayBuiler`.
+        /// </summary>
+        internal const int FilterTextLengthToExcludeExpandedItemsExclusive = 2;
```

Comment:
great comment.  thanks.  

nit: consider making this private (if not actually used anywhere), or 'public' (since we default to that normally in files, knowing that it will have an effective accessibility of 'internal' given the outer type).

Reply:
Sure. Made it public since it's used in tests